src/entities/MovieTitle/MovieTitle.jsx

```diff
@@ -0,0 +1,47 @@
+import "./MovieTitle.css";
+import { FontAwesomeIcon } from "@fortawesome/react-fontawesome";
+import { faEllipsisVertical } from "@fortawesome/free-solid-svg-icons";
+import { useState } from "react";
+
+export const MovieTitle = ({
+  imgUrl,
+  movieName,
+  releaseYear,
+  genres,
+  onClickCB,
+}) => {
+  const [isContextDropdownOpen, setContextDropdownOpen] = useState(false);
+
+  const handleToggleDropdown = () => {
```

Comment:
what is handled here? it's not the best case for this name, it's better for cases like `handleButtonClick`, we talked about it in previous modules. In this case I would just suggest using 
```suggestion
  const toggleDropdown = () => {
```

Reply:
done